### PR TITLE
OHNI Updates

### LINKED ISSUE #636: Ohni Update
We are requesting the Ohni address be updated. Our previous CEO is no longer with us however has the old contract access so we merged contracts. 

We are the proper owners and have the trademark for Ohni.

### PR DESCRIPTION
**Pull request by JonathonDunford**
Closes #636 

This request needs to be reviewed and accepted by other members of the team, but I figured I would put it into a PR. First, read through #636 

@jamesohni Can you provide more desired information for the links and description and what not?
Look at the current file in the `Files Changed` tab.

### CODE REVIEW

File: new/0x6f539a9456a5bcb6334a1a41207c3788f5825207.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+addr: '0x6f539a9456a5bcb6334a1a41207c3788f5825207'
+decimals: 18
+description: >-
+  Ohni is an economic system for the new age.
+links:
+- Website: http://ohni.us
+- Email: mailto:customer.service@ohni.io
```
Comment (jamesohni) [inline new/0x6f539a9456a5bcb6334a1a41207c3788f5825207.yaml]: Info@ohni.us

File: new/0x6f539a9456a5bcb6334a1a41207c3788f5825207.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+- Github: https://github.com/ttmohni/Ohni
+- Reddit: https://www.reddit.com/r/Ohni_Official/
```
Comment (jamesohni) [inline new/0x6f539a9456a5bcb6334a1a41207c3788f5825207.yaml]: We do not have an active reddit at the moment

File: new/0x6f539a9456a5bcb6334a1a41207c3788f5825207.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+addr: '0x6f539a9456a5bcb6334a1a41207c3788f5825207'
+decimals: 18
+description: >-
+  Ohni is an economic system for the new age.
+links:
+- Website: http://ohni.us
+- Email: mailto:customer.service@ohni.io
+- Github: https://github.com/ttmohni/Ohni
```
Comment (jamesohni) [inline new/0x6f539a9456a5bcb6334a1a41207c3788f5825207.yaml]: GitHub.com/OhniCoin

File: new/0x6f539a9456a5bcb6334a1a41207c3788f5825207.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+- Reddit: https://www.reddit.com/r/Ohni_Official/
+- Twitter: https://twitter.com/ohni_io
```
Comment (jamesohni) [inline new/0x6f539a9456a5bcb6334a1a41207c3788f5825207.yaml]: Twitter.com/ohni_official

File: new/0x6f539a9456a5bcb6334a1a41207c3788f5825207.yaml [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+- Telegram: https://t.me/OhniGroup
+- Whitepaper: https://ohni.io/whitepaper.pdf
```
Comment (jamesohni) [inline new/0x6f539a9456a5bcb6334a1a41207c3788f5825207.yaml]: Revising. Will be available at ohni.us/whitepaper.pdf soon

File: tokens/0x7f2176ceb16dcb648dc924eff617c3dc2befd30d.yaml [hunk 6]
```diff
@@ -1,5 +1,5 @@
 ---
-__FORKDELTA_CUSTOM_SYMBOL: OHNI
+__FORKDELTA_CUSTOM_SYMBOL: OHNI_OLD
 addr: '0x7f2176ceb16dcb648dc924eff617c3dc2befd30d'
 decimals: 0
 description: >-
```
Comment (jamesohni) [inline tokens/0x7f2176ceb16dcb648dc924eff617c3dc2befd30d.yaml]: "THIS IS THE LEGACY OHNI TOKEN, PLEASE DO NOT USE"

File: new/0x6f539a9456a5bcb6334a1a41207c3788f5825207.yaml [hunk 7]
```diff
@@ -0,0 +1,15 @@
+---
+addr: '0x6f539a9456a5bcb6334a1a41207c3788f5825207'
+decimals: 18
+description: >-
+  Ohni is an economic system for the new age.
```
Comment (jamesohni) [inline new/0x6f539a9456a5bcb6334a1a41207c3788f5825207.yaml]: Ohni is an economic system for consumers, by consumers.

**Comment by jamesohni** [comment]
I think I did what you requested. Let me know if you need anything else

**Comment by Kryptoniq** [comment]
@JonathonDunford this looks legit AFAIK.
My take is; we should give an ultimatum, if we don't hear from the former CEO in X number of days, the changes should be processed.

**Comment by JonathonDunford** [comment]
@Kryptoniq I wouldn't even know how to get in touch with the former CEO.

**Comment by Kryptoniq** [comment]
@JonathonDunford if he is actually serious with the project then should  come out.
I see the team's vision. They want to move forward.

**Comment by jamesohni** [comment]
This is one of the main reasons our previous CEO is gone. He kept freezing and freezing and freezing left and right. We do not freeze anyone. The only reason we have it is to comply with the ERC20 standard token. Many cryptocurrencys have freezing in their contract. I would say at least 50% listed on FD have that.

---- On Sat, 24 Mar 2018 05:43:59 -0700 notifications@github.com wrote ----

@Kryptoniq commented on this pull request.

I'm against the account freezing function on the smart contract.
From a bitcointalk user on the Ohni thread.
Quote from: mithras on October 16, 2017, 10:51:01 AM Why should any exchange list your token (which is needed for most people to be interested and for it to gain any real value), when you have screwed the one exchange that did by freezing the wallet? https://bitcointalk.org/index.php?topic=2275733.msg23081794#msg23081794

Also, a former team member complained that the CEO threatened to freeze his account if he doesn't comply to some terms.
https://bitcointalk.org/index.php?topic=2275733.msg23298435#msg23298435

Why should we trust the new team with this function? They can simply decide to lock anyone's account at anytime and this is toxic to the Ethereum ecosystem.

**Comment by Kryptoniq** [comment]
@jamesohni  I don't think that 50% of tokens on FD have the account freezing function. Not even 10%. Except you can provide proofs.

We do care about investors and we don't want previous occurrences to affect anyone in the future.

**Comment by jamesohni** [comment]
Ok. One of the main reasons the old CEO is gone is due to him freezing many people like I said.

The only reason we have it in the contract is due to the fact the contract needs to be the same to be able to copy the tokens. Also, freezing is in the standard ERC20 token.

---- On Sat, 24 Mar 2018 08:06:00 -0700 notifications@github.com wrote ----

@jamesohni I don't think that 50% of tokens on FD have the account freezing function. Not even 10%. Except you can provide proofs.

We do care about investors and we don't want previous occurrences to affect anyone in the future.

**Comment by jamesohni** [comment]
We guarantee to not freeze anyone. It was required in the contract to make the merge work. 

I am on your side. I am against freezing. The previous CEO was freezing all the time.

We are even offering people Ohni on the new contract if there old ohni is frozen.

**Comment by Kryptoniq** [comment]
@jamesohni AFAIK, you can do a token swap without replicating the functions of the previous contract.
I'd like to hear from @JonathonDunford.

**Comment by freeatnet** [comment]
See #636.

**Pull request closed without merging**